core/services/ocr2/plugins/ocr2keeper/evm21/encoder_test.go

```diff
@@ -160,17 +160,6 @@ func TestEVMAutomationEncoder21_Encode_errors(t *testing.T) {
 		assert.Equal(t, err.Error(), "tx hash parse error: empty hex string")
 	})
 
-	t.Run("an invalid upkeep type causes an error", func(t *testing.T) {
```

Comment:
This test is no longer valid as now UpkeepPayload cannot be generated with an invalid upkeep type (previously it was possible)